Gruntfile.js

```diff
@@ -9,7 +9,7 @@ module.exports = function (grunt) {
   grunt.loadNpmTasks('grunt-contrib-jshint');
   grunt.loadNpmTasks('grunt-contrib-concat');
   grunt.loadNpmTasks('grunt-contrib-watch');
-  grunt.loadNpmTasks('grunt-contrib-uglify');
```

Comment:
this is a problem - uglify isn't used in dev mode but to compile for production its necessary - why did you need to take it out?